### PR TITLE
Fix (CTV-1745): Stop video stream video player so that ads can buffer videos

### PR DESCRIPTION
**Pull request by tonyr59h**
Didn't catch this originally because we had been testing on the Realtor ad, which has no video. This PR makes sure to set the video control to `stop` before starting up the TruexAdRenderer. Tested the midrolls in the reference app 👍

### CODE REVIEW

File: components/ContentFlow.brs [hunk 1]
```diff
@@ -228,6 +228,8 @@ sub onTruexAdDataReceived(event as object)
     m.adRenderer.action = tarInitAction
 
     ? "TRUE[X] >>> ContentFlow::onTruexAdDataReceived() - starting TruexAdRenderer..."
+    ' CTV-1745 - must stop the video player since Roku only allows one Video stream to buffer
+    m.videoPlayer.control = "stop"
```
Comment (episcopus) [inline components/ContentFlow.brs:232]: Do we not need any resume logic on this? What about the handling of ad skipping? Does the seeking logic work with the video player stopped?

**Review by episcopus — COMMENTED** [review]
I just want to make sure we tested this and confirmed no other changes were needed to record the play head and restart it from that location when resuming the stream after stopping.

File: components/ContentFlow.brs (hunk 1 above)
Comment (tonyr59h) [reply components/ContentFlow.brs:232]: The resume/seeking logic is already implemented and seems to be working during my smoke test.

When the ad break begins [the stream position is saved.](https://github.com/socialvibe/truex-roku-google-ad-manager-reference-app/blob/master/components/ContentFlow.brs#L203)
When the engagement ends [the stream is played from that position](https://github.com/socialvibe/truex-roku-google-ad-manager-reference-app/blob/master/components/ContentFlow.brs#L355) plus the length of the ad break.

Comment (episcopus) [reply components/ContentFlow.brs:232]: ah, thanks for confirming nice - makes this change very simple

**Review by episcopus — APPROVED** [review]
lgtm

**Pull request merged**